dynamic_rest/serializers.py

```diff
@@ -301,6 +317,29 @@ def get_name(cls):
         return cls.Meta.name
 
     @classmethod
+    def get_url(self, pk=None):
+        # if associated with a registered viewset, use its URL
+        url = getattr(self, '_url', None)
```

Comment:
Got it. So this more like the "canonical URL for the serializer" as opposed to "canonical URL for the model". That makes sense 👍